spec/controllers/registrations_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe RegistrationsController, type: :controller do
```

Comment:
コントローラー名は正しく書かなければならないので↓のようになります。

```suggestion
RSpec.describe Users::RegistrationsController, type: :controller do
```